openfecwebapp/api_caller.py

```diff
@@ -253,29 +253,29 @@ def landing_mock_data():
         }
     }
 
-def load_top_candidates(sort):
+def load_top_candidates(sort, office=None, cycle=2016, per_page=5):
```

Comment:
Nice work parameterizing these values with the defaults!  Same for the changes below.